specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-10-01-preview/examples/BackupInstanceOperations/PutBackupInstance.json

```diff
@@ -0,0 +1,150 @@
+{
+  "parameters": {
+    "subscriptionId": "04cf684a-d41f-4550-9f70-7708a3a2283b",
+    "resourceGroupName": "000pikumar",
+    "vaultName": "PratikPrivatePreviewVault1",
+    "api-version": "2021-07-01",
+    "backupInstanceName": "testInstance1",
+    "parameters": {
+      "properties": {
+        "objectType": "BackupInstance",
+        "friendlyName": "harshitbi2",
+        "dataSourceSetInfo": {
+          "objectType": "DatasourceSet",
+          "resourceID": "/subscriptions/f75d8d8b-6735-4697-82e1-1a7a3ff0d5d4/resourceGroups/viveksipgtest/providers/Microsoft.DBforPostgreSQL/servers/viveksipgtest",
+          "resourceName": "viveksipgtest",
+          "resourceType": "Microsoft.DBforPostgreSQL/servers",
+          "resourceUri": "",
+          "resourceLocation": "",
```

Comment:
Do resourceProperties(instanceUuid) need not be specified here, in Validate.json and other places as well?

Reply:
This workload specific field while examples are workload agnostics.